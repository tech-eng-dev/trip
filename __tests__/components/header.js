import React from 'react';
import renderer from 'react-test-renderer';
import CustomHeader from '../../src/components/header';


test('Header renders correctly', () => {
  const header = renderer.create(<CustomHeader
    onPressLeft={() => {}}
    onPressRight={() => {}}
    left="ios-arrow-back"
    right="md-add"
    title="Title"
    backgroundColor="black"
  />).toJSON();
  expect(header).toMatchSnapshot();
});
