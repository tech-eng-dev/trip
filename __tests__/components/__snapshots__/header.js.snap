// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<Component
  style={
    Object {
      "backgroundColor": "black",
    }
  }
>
  <Component>
    <Component
      onPress={[Function]}
      testID="Title-header-left-button"
      transparent={true}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#F5FAFE",
              "fontSize": 30,
            },
            Object {
              "marginHorizontal": 6,
            },
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        
      </Text>
    </Component>
  </Component>
  <Component>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Object {
          "fontSize": 18,
          "fontWeight": "bold",
          "textAlign": "center",
          "width": 500,
        }
      }
    >
      Title
    </Text>
  </Component>
  <Component>
    <Component
      onPress={[Function]}
      testID="header-right-button"
      transparent={true}
    >
      <Text
        accessible={true}
        allowFontScaling={false}
        ellipsizeMode="tail"
        style={
          Array [
            Object {
              "color": "#F5FAFE",
              "fontSize": 30,
            },
            Object {
              "marginHorizontal": 6,
            },
            Object {
              "fontFamily": "Ionicons",
              "fontStyle": "normal",
              "fontWeight": "normal",
            },
          ]
        }
      >
        
      </Text>
    </Component>
  </Component>
</Component>
`;
